gocourse05/zooobservation/animal/animal_test.go

```diff
@@ -0,0 +1,70 @@
+package animal
+
+import (
+	"testing"
+
+	"zooobservation/camera"
+	"zooobservation/common"
+)
+
+func TestAnimal_Move(t *testing.T) {
+	direct := common.Left
```

Comment:
`direct` та `direction` - це в загальному різні слова:
```suggestion
	direction := common.Left
```

https://www.merriam-webster.com/dictionary/direct
https://www.merriam-webster.com/dictionary/direction